Advanced-React-Patterns/course-notes.md

```diff
@@ -0,0 +1,114 @@
+## Design Patterns
```

Comment:
Please add in the description of the PR the title of the course and modules, thanks in advanced, the rest looks good